tests/get_or_create/models.py

```diff
@@ -32,6 +32,14 @@ class Thing(models.Model):
     name = models.CharField(max_length=255)
     tags = models.ManyToManyField(Tag)
 
+    @property
+    def capitalized_name_property(self):
+        return self.name
+
+    @capitalized_name_property.setter
+    def capitalized_name_property(self, val):
+        self.name = val.capitalize()
```

Comment:
I think the suggestion is to maintain the "Invalid field name(s) for model ..." error message a property that doesn't have a `setter`.